softwarereview_editor_management.pt.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+aliases: editorialmanagement.html
+---
+
+# Gerenciamento editorial {#editorialmanagement}
+
+```{block, type="summaryblock"}
+Orientação para gerenciar a equipe editorial.
+```
+
+## Recrutamento de novos editores {#recruiting-new-editors}
+
+O recrutamento de novos editores e a manutenção de um conselho editorial suficiente e equilibrado é responsabilidade da [Líder de revisão de software](https://ropensci.org/about/#team) com o apoio e a orientação do conselho editorial.
+
+Etapas:
+
+- Inicie um canal privado para discussão (para que você não tenha um histórico no canal de editores no qual futuros editores entrarão, o que pode ser incômodo).
+
+- Faça ping nos editores para garantir que eles recebam uma notificação, pois esse é um tópico importante.
```

Comment:
'Faça ping nos editores' poderia ser mudado para 'Marque (ping) editores'

Reply:
Suggestions included in the review.